src/styles/themes/common.js

```diff
@@ -0,0 +1,36 @@
+export const typography = {
+  // Enable v2 variants.
+  // https://material-ui.com/style/typography/#migration-to-typography-v2
+  useNextVariants: true,
+
+  fontFamily: 'Noto Sans',
+
+  h1: { fontFamily: 'Montserrat', fontWeight: 700 },
+  h2: { fontFamily: 'Montserrat', fontWeight: 700 },
+  h3: { fontFamily: 'Montserrat', fontWeight: 700 },
+  h4: { fontFamily: 'Montserrat', fontWeight: 600 },
+  h5: { fontFamily: 'Montserrat', fontWeight: 600 },
+  h6: { fontFamily: 'Montserrat', fontWeight: 700 },
+
+  body1: {
+    fontFamily: 'Libre Baskerville',
+    fontSize: '1rem',
+    lineHeight: '1.5'
+  },
+
+  button: {
+    letterSpacing: 0.25,
+    textTransform: 'none'
+  },
+
+  overline: {
+    letterSpacing: 0.25,
+    textTransform: 'none'
+  }
+}
+
+export const errorStateColours = {
+  palette: {
+    error: { main: '#990000', border: '#d8352a' }
```

Comment:
At the risk of prolonging a super long-running PR, and with apologies if this has been suggested, but how would it be to extract the hard-coded colour names to a file of constants? It would be great if they had semantic names rather than just being "magic".

Reply:
They could be an export from this very file in fact.

Reply:
> It would be great if they had semantic names

What would be good names?

Reply:
> What would be good names?

If I read it right, e.g. `#990000` is colour "Core", strength 700. So, maybe `colours.core700` or something?

Reply:
HMMMM yeah, that feels pretty good... I'll follow the https://material-ui.com/customization/color/#color style... so core: {"700": "#990000"}?

Something like the example below?

```
import purple from '@material-ui/core/colors/purple';
import red from '@material-ui/core/colors/red';

const primary = red[500]; // #F44336
const accent = purple['A200']; // #E040FB
const accent = purple.A200; // #E040FB (alternative method)
```

Reply:
Yeah, that sounds good. As long as you use the same names as the style guide, so, e.g. 'core', 'accent', etc.

Reply:
Done 🎉